### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by nichtich** [pg-format/pg-formatter#10] [General]
Hi! I'd like to take the chance and briefly introduce myself and my interest in the project. My name is Jakob Voß, I work in research & development at [a library union service center](https://www.wikidata.org/wiki/Q27863572) in Germany where we have to deal with data in diverse formats. My profession is library and information science and computer science. My [PhD thesis](http://aboutdata.org/) was about data formats in general and I manage a [collection of data format](https://format.gbv.de/). This includes a lot of [data structuring languages](https://format.gbv.de/structure) such as JSON, YAML, RDF... When I started to include languages for graphs such as GraphML, I stumbled upon PG format as introduced in https://arxiv.org/abs/1907.03936. Writing [a tool to convert between property graph formats](https://github.com/gbv/pgraph?tab=readme-ov-file#pgraph) included slightly adjusting PG format to get a full and final specification. The format and its data model are great for data exchange and we will use it for our work with knowledge graphs but the devil is in the details. Thanks for your great work and collaboration!

**Comment by hchiba1** [comment]
Thank you for your contribution!
This presents a great opportunity for us to revisit and refine the details, including data types and other specifications.

> the devil is in the details

Exactly what I think!
